src/core/cosmosdb_containers.tf

```diff
@@ -0,0 +1,16 @@
+#
+# LolliPOP containers
+#
+resource "azurerm_cosmosdb_sql_container" "lollipop_thumbprints" {
+
+  name                = "lollipop-thumbprints"
```

Comment:
```suggestion
  name                = "lollipop-pubkeys"
```

Reply:
did in https://github.com/pagopa/io-infra/pull/437/commits/45f1a723e3ae41eb2cc7b0fea76aef7bc2536b69